homeassistant/components/iskra/coordinator.py

```diff
@@ -37,11 +37,6 @@ def __init__(self, hass: HomeAssistant, device: Device) -> None:
             update_interval=update_interval,
         )
 
-    @property
-    def available(self):
-        """Return True if entity is available."""
-        return self._is_available
-
     @property
     def device(self):
```

Comment:
Just store the variable in `self.device`, then you can remove the extra property

Reply:
done